src/EventSubscriber/IconOptionsSubscriber.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_paragraphs\EventSubscriber;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Drupal\oe_paragraphs\Event\IconOptionsEvent;
+
+/**
+ * Provides options for the icon field.
+ */
+class IconOptionsSubscriber implements EventSubscriberInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function getSubscribedEvents(): array {
+    return [
+      IconOptionsEvent::class => 'getIconOptions',
+    ];
+  }
+
+  /**
+   * Gets the icon options.
+   *
+   * @param \Drupal\custom_events\Event\IconOptionsEvent $event
+   *   Allowed format event object.
+   */
+  public function getIconOptions(IconOptionsEvent $event): void {
+    $event->setIconOptions([
+      'arrow-down' => 'Arrow down',
+      'external' => 'External',
+      'arrow-up' => 'Arrow up',
+      'audio' => 'Audio',
+      'book' => 'Book',
+      'breadcrumb' => 'Breadcrumb',
+      'brochure' => 'Brochure',
+      'budget' => 'Budget',
+      'calendar' => 'Calendar',
+      'camera' => 'Camera',
+      'check' => 'Check',
+      'close' => 'Close',
+      'close-dark' => 'Close dark',
```

Comment:
```suggestion
      'close-dark' => t('Close dark'),
```
Should we translate labels since we can't foresee all cases how event can be invoked and used?

Reply:
It's our event, so we control where it's invoked. The real reason to add t() is that a website could have the backend translated in another language and, from what I've seen, passing the values in t() is the only way to translate these.
@imanoleguskiza will check if this is the case, and fix it.

Reply:
Indeed, since its a method providing the options this are not translatable as configuration translation, so we need to provide the t() as the only alternative.